ehr/resources/views/groupOverlaps.html

```diff
@@ -154,16 +154,19 @@
                 renderTo: this.qwpDiv.id,
                 schemaName: 'study',
                 queryName: queryName,
-                title: 'Group Members'
+                title: 'Group Members',
             };
 
             if (!start){
                 Ext4.Msg.alert('Error', 'Must provide a start date');
                 return;
             }
 
+            // By Kollil, 3/22/2018 : In 17.2 the filter is not working. Removed the the line 168 and added the filterArray at line 169.
+            // The Animal group search is working fine now after this change.
             if (groupName)
-                config.removeableFilters = [LABKEY.Filter.create('groupId/name', groupName, LABKEY.Filter.Types.EQUAL)];
+                //config.removeableFilters = [LABKEY.Filter.create('groupId/name', groupName, LABKEY.Filter.Types.EQUAL)];
```

Comment:
isnt userFilters the new name for removeableFilters?  this change prevents the user from removing the group name filter, doesnt it?

Reply:
I believe that is the intention given that user enters filter criteria in a form, which then results in a filtered grid in a different panel, having removable filters on this resultant grid would be confusing.

Reply:
I think it's clear that if there was a bug preventing the filter from being applied, that's a problem.

I can see the argument on there being a filter panel above, but by that same logic then shouldnt start/end date also be immutable? It is possible for the top filter fields and DR to get out of sync.  

This is a small enough change that it's probably not a big deal, but in general I would tend toward preserving capabilities for the user. For example, what is they want to do another filter type?  More than one group?  Contains? etc.